etcdserver/api/v3rpc/rpctypes/error.go

```diff
@@ -24,6 +24,7 @@ var (
 	ErrGRPCEmptyKey     = grpc.Errorf(codes.InvalidArgument, "etcdserver: key is not provided")
 	ErrGRPCKeyNotFound  = grpc.Errorf(codes.InvalidArgument, "etcdserver: key not found")
 	ErrGRPCValue        = grpc.Errorf(codes.InvalidArgument, "etcdserver: value is provided")
+	ErrGRPCLease        = grpc.Errorf(codes.InvalidArgument, "etcdserver: lease is provided")
```

Comment:
no strong opinion either way, just so long as the value/lease error naming is consistent